### PR TITLE
Update to 2.2

### PR DESCRIPTION
**Pull request by cho2**
Update to 2.2 based on https://github.com/flathub/net.poedit.Poedit/pull/4

### CODE REVIEW

File: net.poedit.Poedit.json [hunk 1]
```diff
@@ -1,21 +1,21 @@
 {
     "app-id": "net.poedit.Poedit",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.26",
+    "runtime-version": "3.28",
```
Comment (bilelmoussaoui) [inline net.poedit.Poedit.json]: Why not 3.30?

Comment (cho2) [reply net.poedit.Poedit.json]: Build failed after update runtime to 3.30

```
...
configure: error: Package requirements (glib-2.0 gtk+-3.0 enchant) were not met:
Package 'enchant', required by 'virtual:world', not found
...
```

Comment (bilelmoussaoui) [reply net.poedit.Poedit.json]: You need to add enchant as it was removed from freedesktop 18.08

Comment (bilelmoussaoui) [reply net.poedit.Poedit.json]: Yeah, that should be done too, not sure if the app requires anything that isn't already on 18.08.

Comment (cho2) [reply net.poedit.Poedit.json]: Any idea to build `enchant`? 
Just take a look at https://gitlab.gnome.org/GNOME/recipes/blob/master/flatpak/org.gnome.Recipes.json and https://gitlab.gnome.org/GNOME/geary/blob/master/org.gnome.Geary.json, i have no clue.

Comment (bilelmoussaoui) [reply net.poedit.Poedit.json]: You have misclosed the wxwidgets module. You can just open the json file in any text editor with syntax highlithing to see the issue :+1: 
Thanks for taking care of updating it.

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by cho2** [comment]
```
configure: error: Package requirements (glib-2.0 gtk+-3.0 enchant) were not met:
Package 'enchant', required by 'virtual:world', not found
Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.
Alternatively, you may set the environment variables GTK_SPELL3_CFLAGS
and GTK_SPELL3_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```

File: net.poedit.Poedit.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+    "runtime-version": "3.30",
     "sdk": "org.gnome.Sdk",
     "command": "poedit",
     "finish-args": [
         "--share=ipc",
         "--socket=x11",
         "--filesystem=home",
-        /* Network */ 
+        /* LibSecret*/
+        "--talk-name=org.freedesktop.secrets",
```
Comment (bilelmoussaoui) [inline net.poedit.Poedit.json]: Why this was added btw?

**Comment by bilelmoussaoui** [comment]
bot, build

File: net.poedit.Poedit.json (hunk 2 above)
Comment (cho2) [reply net.poedit.Poedit.json]: it based on your previous work https://github.com/flathub/net.poedit.Poedit/blob/43b5947c972f06a6aaf0a41ff263199707db3a31/net.poedit.Poedit.json

**Comment by bilelmoussaoui** [comment]
bot, build

I built it locally and everything works as expected :+1:

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by cho2** [comment]
thanks @bilelmoussaoui :+1:

**Comment by bilelmoussaoui** [comment]
Will merge once the latest test build here ends https://flathub.org/builds/#/builders/2/builds/7533

**Pull request merged**